static/bidder-info/indicue.yaml

```diff
@@ -0,0 +1,9 @@
+aliasOf: adtelligent
+endpoint: "http://ghb.console.indicue.com/pbs/ortb"
+maintainer:
+  email: "ops@indicue.com"
```

Comment:
PBS team has sent email to verfify above address. Please respond back with "received" response

Reply:
received